### PR TITLE
Revise

### PR DESCRIPTION
**Pull request by wdotte**

### CODE REVIEW

**Comment by wdotte** [comment]
pls check my debug hint

File: webtimeserver/webtimeserver.h [hunk 1]
```diff
@@ -12,6 +12,7 @@
 #include <signal.h>
 #include <time.h>
 
+// Undefine DBP if no debug messages desired
```
Comment (wolfdieterotte) [inline webtimeserver/webtimeserver.h:15]: That seems fine to me

Comment (wdotte) [reply webtimeserver/webtimeserver.h:15]: okay, than please merge

**Comment by wolfdieterotte** [comment]
don't know

**Comment by wdotte** [comment]
come on, you can to it

**Comment by wolfdieterotte** [comment]
okay, will do

**Pull request closed without merging**